quickstart/python/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Use the official Python image.
+# https://hub.docker.com/_/python
+FROM python:3.7
+
+# Copy local code to the container image.
+ENV APP_HOME /app
+WORKDIR $APP_HOME
+COPY . .
+
+# Install production dependencies.
+RUN pip install Flask gunicorn
```

Comment:
This is fine for this app, but the convention is to include a requirements.txt file with all dependencies (including versions if desired), and then run pip install -r requirements.txt instead of this.

Reply:
The reason this was done is so that the users can copy-paste just two files (the `Dockerfile`  and `app.py`) to  get the  app running. What if I add a comment like  this?

```
# NOTE: This is for simplicity. Convention is to use a requirements.txt in your
# app and run "pip install -r requirements.txt" in the Dockerfile instead.
```